custom_components/better_thermostat/model_fixes/SEA801-Zigbee_SEA802-Zigbee.py

```diff
@@ -1,7 +1,9 @@
 def fix_local_calibration(self, entity_id, offset):
     # device SEA802 fix
-    if offset > -1.5:
-        offset -= 1.5
+    if (self.cur_temp - 0.25) <= self.bt_target_temp:
```

Comment:
this will also always on heating do -2.5, shouldn't we have something, that just corrects it if it would be lower than -2.5? for example if the calibration mode is calculating -4 then we would have in the end -6.5.

Reply:
someone told me the SEA, needs this -2.5 to get to heating, so yes, it shoud be added every time

Reply:
i'll push a little change as well